3rdparty/exported/quickjs/quickjs.c

```diff
@@ -48,6 +48,8 @@
 #include "libbf.h"
 #endif
 
+#include "quickjs-time.h"
```

Comment:
Does this modify the actual QuickJS source code? This seems like something we'll forget to do when we next upgrade QuickJS?

Reply:
Yes, and yes, but there are only so many solutions to this problem and we don't update quickjs often, since it's not released often.